code/Test_definitions/connected-network-type-subscriptions.feature

```diff
@@ -1,289 +1,489 @@
     And the response property "$.message" contains a user friendly text
 
-  @connected_network_type_subscriptions_19_permission_denied
-  Scenario: subscription creation with inconsistent access token for requested events subscription
+  @connected_network_type_subscriptions_delete_401.9_invalid_access_token
+  Scenario: Invalid access token
+    Given the header "Authorization" is set to an invalid access token
+    And use BaseUrL
+    When the request "deleteConnectedNetworkTypeSubscription" is sent
+    Then the response header "Content-Type" is "application/json"
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED" or "AUTHENTICATION_REQUIRED"
+    And the response property "$.message" contains a user friendly text
+
+  
+##################
+# Error code 403
+##################
+
+  @connected_network_type_subscriptions_create_403.1_permission_denied
+  Scenario: subscription creation without having the required scope
     # To test this, a token does not have the required scope
-    Given a valid subscription request body
-    And the request body property "$.device" is set to a valid testing device supported by the service
-    And header "Authorization" set to access token referring different scope
+    Given header "Authorization" set to access token referring different scope
+    And use BaseUrL
     When the request "createConnectedNetworkTypeSubscription" is sent
     Then the response property "$.status" is 403
     And the response property "$.code" is "PERMISSION_DENIED"
     And the response property "$.message" contains a user friendly text
 
-  @connected_network_type_subscriptions_20_unnecessary_identifier
-  Scenario: subscription creation with both a 3-legged token and explicit device identifier
-    # This test applies whether the device associated with the access token matches the explicit device identifier or not
-    # For 3-legged access tokens, an explicit device identifier MUST NOT be provided
+  @connected_network_type_subscriptions_create_403.2_invalid_token_context
+  Scenario: subscription creation with invalid access token context for requested events subscription
+    # To test this, a token does not have the required device identifier
     Given a valid subscription request body
+    And use BaseUrL
     And the request body property "$.device" is set to a valid testing device supported by the service
-    And header "Authorization" set to access token also referring to a device (which may or may not be the same device)
+    And header "Authorization" set to access token referring different device
     When the request "createConnectedNetworkTypeSubscription" is sent
-    Then the response property "$.status" is 422
-    And the response property "$.code" is "UNNECESSARY_IDENTIFIER"
+    Then the response property "$.status" is 403
+    And the response property "$.code" is "INVALID_TOKEN_CONTEXT"
```

Comment:
3-legged token + explicit device identifier is always `422 UNNECESSARY_IDENTIFIER`
```suggestion
    Then the response property "$.status" is 422
    And the response property "$.code" is "UNNECESSARY_IDENTIFIER"
```